src/index.js

```diff
@@ -13,8 +13,12 @@ const ES6_BROWSER_EMPTY = '\0node-resolve:empty.js';
 // which deploy both ESM .mjs and CommonJS .js files as ESM.
 const DEFAULT_EXTS = [ '.mjs', '.js', '.json', '.node' ];
 
+const existsAsync = file => new Promise(fulfil => fs.exists(file, exists => fulfil(exists)));
```

Comment:
Minor point: This could be `...fs.exists(file, fulfil))`

Reply:
fixed